plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -30,7 +30,7 @@ import kotlin.properties.Delegates
  * However, it doesn't have to be the UI thread.
  */
 
-internal class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
+class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
```

Comment:
em.. we make this class public, because we are referencing this class in the sdk module directly I guess?

Reply:
Yes, exactly as __all__ other plugins 😄